src/components/FeaturedVideos/FeaturedVideos.jsx

```diff
@@ -30,7 +29,7 @@ const FeaturedVideos = ({ page = "" }) => {
         ) : null}
       </div>
       <div className={styles.featuredVideos}>
-        {videoList.map((video, i) => (
+        {videos.map((video, i) => (
           <MainVideoCard key={i} video={video} />
```

Comment:
since your backend database is complete, you can use `video._id` as your key here

Reply:
Will correct this in the next PR. Thanks for pointing it out!